src/Altinn.Platform/Altinn.Platform.Authorization/IntegrationTests/PolicyRepositoryTest.cs

```diff
@@ -0,0 +1,271 @@
+using System;
```

Comment:
I think this is more like unit test than integration test. So probably move it to a unit test project